### PR TITLE
[Doc] Document Alluxio does not sync metadata from S3 by default [skip ci]

### LINKED ISSUE #7079: [DOC] Document the Alluxio sync is disable.
## Report incorrect documentation

**Location of incorrect documentation**
`getting-started-alluxio.md`

**Describe the problems or issues found in the documentation**
This is from https://github.com/NVIDIA/spark-rapids/issues/6779#issuecomment-1308073009

**Suggested fix for documentation**
Document the behavior of alluxio in our docs. Mention sync is disabled by default so it won't pick up changed files. If you want to enable sync it has performance impact and we need to document where and how it works.

### PR DESCRIPTION
**Pull request by res-life**
Closes #7079

This is a follow-up PR
Document Alluxio does not sync metadata from S3 by default

Signed-off-by: Chong Gao <res_life@163.com>

### CODE REVIEW

File: docs/get-started/getting-started-alluxio.md [hunk 1]
```diff
@@ -419,4 +419,13 @@ See the master.log and worker.log in this path.
 
 ### Auto start Alluxio the master and workers
 After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
-Systemd service can automatically start process if process is terminated.
\ No newline at end of file
+Systemd service can automatically start process if process is terminated.
+
+## Alluxio limitations
+### Alluxio does not sync metadata from UFS(e.g. S3) by default
+Alluxio does not sync metadata from S3 by default, so it won't pick up changed files.
```
Comment (tgravescs) [inline docs/get-started/getting-started-alluxio.md:426]: I'd also like to see a sentence at the very beginning like line 19 that says something like:  sets up Alluxio to handle reads and by default does not handle file updates in the remote blob store. 

We should also explicitly put somethign in here about writes don't go through alluxio.

**Comment by res-life** [comment]
build

File: docs/get-started/getting-started-alluxio.md [hunk 2]
```diff
@@ -419,4 +420,13 @@ See the master.log and worker.log in this path.
 
 ### Auto start Alluxio the master and workers
 After installing Alluxio master and workers, it's better to add a systemd service for each process of master and workers.
-Systemd service can automatically start process if process is terminated.
\ No newline at end of file
+Systemd service can automatically start process if process is terminated.
+
+## Alluxio limitations
+### Alluxio does not sync metadata from UFS(e.g. S3) by default
+Alluxio does not sync metadata from S3 by default if writes don't go through alluxio, so it won't pick up changed files generated by external systems.
```
Comment (tgravescs) [inline docs/get-started/getting-started-alluxio.md:427]:
```suggestion
Alluxio does not sync metadata from S3 by default so it won't pick up any changed files.
```

File: docs/get-started/getting-started-alluxio.md [hunk 3]
```diff
@@ -419,4 +420,13 @@ See the master.log and worker.log in this path.
...
+For example:
+If you update a file in the S3 from 1M size to 10M size and Alluxio already cached the 1M size file, Alluxio cluster
+will always use the 1M file.
+If you want to enable sync it has performance impact.
```
Comment (tgravescs) [inline docs/get-started/getting-started-alluxio.md:431]:
```suggestion
If you want to enable sync it has performance impact which will affect the read performance.
```

File: docs/get-started/getting-started-alluxio.md (hunk 1 above)
Comment (res-life) [reply docs/get-started/getting-started-alluxio.md]: Done

File: docs/get-started/getting-started-alluxio.md (hunk 2 above)
Comment (res-life) [reply docs/get-started/getting-started-alluxio.md:427]: Done

File: docs/get-started/getting-started-alluxio.md (hunk 3 above)
Comment (res-life) [reply docs/get-started/getting-started-alluxio.md:431]: Done

**Comment by res-life** [comment]
build

**Review by tgravescs — APPROVED** [review]

**Pull request merged**